src/transformers/models/layoutlmv2/feature_extraction_layoutlmv2.py

```diff
@@ -220,3 +235,52 @@ def __call__(
             encoded_inputs["boxes"] = boxes_batch
 
         return encoded_inputs
+
+    def post_process(self, outputs, target_sizes, offset_mapping, bbox):
+        """
+        Converts the output of :class:`~transformers.LayoutLMv2ForTokenClassification` into the format expected by the
+        COCO api. Only supports PyTorch.
+
+        Args:
+            outputs (:obj:`Dict`):
+                Raw outputs of the model.
+            target_sizes (:obj:`torch.Tensor` of shape :obj:`(batch_size, 2)`):
+                Tensor containing the original size (h, w) of each image of the batch.
+            offset_mapping (:obj:`torch.Tensor` of shape :obj:`(batch_size, x, 2)`):
+                Tensor coming from the "offset_mapping" field of the outputs of
+                :class:`~transformer.LayoutLMv2TokenizerFast`.
+            bbox (:obj:`torch.Tensor` of shape :obj:`(batch_size, x, 4)`):
+                Tensor coming from the "bbox" field of the outputs of :class:`~transformer.LayoutLMv2TokenizerFast`.
+
+        Returns:
+            :obj:`List[Dict]`: A list of dictionaries, each dictionary containing the scores, labels and boxes for an
+            image in the batch as predicted by the model.
+        """
+        out_logits = outputs.logits
+
+        assert len(out_logits) == len(
+            target_sizes
+        ), "Make sure that you pass in as many target sizes as the batch dimension of the logits"
+        assert (
+            target_sizes.shape[1] == 2
+        ), "Each element of target_sizes must contain the size (h, w) of each image of the batch"
+
+        prob = nn.functional.softmax(out_logits, -1)
+        scores, labels = prob[..., :-1].max(-1)
+        scores = scores.tolist()
+        labels = labels.tolist()
+        boxes = bbox.tolist()
+        target_sizes = target_sizes.tolist()
+        offsets = offset_mapping.tolist()
+
+        results = []
+
+        for s, l, b, o, (height, width) in zip(scores, labels, boxes, offsets, target_sizes):
+            # only keep start of a particular word (i.e. offset start == 0) (e.g. [San, Fran, ##cis, ##co] -> [San])
```

Comment:
s, l, b, o aren't that easily readable, could we use something more informative instead?

Also, for [San, Fran, ##cis, ##co] -> [San], we are actually using the predictions for both [San] and [Fran], as they are both tokens that are at the start of a particular word.
```suggestion
        for s, l, b, o, (height, width) in zip(scores, labels, boxes, offsets, target_sizes):
            # only keep start of a particular word (i.e. offset start == 0) (e.g. [San, Fran, ##cis, ##co] -> [San, Fran])
```